.github/workflows/stale.yml

```
@@ -0,0 +1,40 @@
+name: 'Close inactive issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+jobs:
+  close-issues-and-pull-requests:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          days-before-issue-stale: 90
+          days-before-issue-close: 30
+          stale-issue-label: 'stale'
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not
+            had any activity within 90 days. It will be automatically closed if no
+            further activity occurs within 30 days. Leave a comment if
+            you feel this issue should remain open. Thank you!
+          close-issue-message: >
+            This issue has been automatically closed due to inactivity. Please
+            feel free to reopen if you feel it is still relevant. Thank you!
+
+          days-before-pr-stale: 90
+          days-before-pr-close: 30
+          stale-pr-label: 'stale'
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it
+            has not had any activity within 90 days. It will be automatically
+            closed if no further activity occurs within 30 days. Leave a comment
+            if you feel this pull request should remain open. Thank you!
+          close-pr-message: >
+            This pull request has been automatically closed due to inactivity.
+            Please feel free to reopen if you intend to continue working on it.
+            Thank you!
```

@simon-mo this change should address both of your comments from https://github.com/vllm-project/vllm/issues/9435#issuecomment-2418137303

```suggestion
            Thank you!
          exempt-draft-pr: true
          # If an inactive issue/pr becomes active, keep it open
          labels-to-add-when-unstale: 'keep-open'
          exempt-issue-labels: 'keep-open'
          exempt-pr-labels: 'keep-open'

```

---

So a single comment on a stale issue would then keep it open forever? I’m not so sure about the “labels to add” option unless I’m misunderstanding it. 

The other 3 look good! Thanks

---

To allow anyone to add the label, I propose a custom workflow that responds to a comment with a “/keep-open” command. I’ll add that to this PR.

---

actually, will continue the discussion on labeling control in the RFC issue.